alluxio/pom.xml

```diff
@@ -68,15 +68,27 @@
             <artifactId>alluxio-minicluster</artifactId>
             <version>${alluxio.version}</version>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.hadoop</groupId>
+                    <artifactId>hadoop-client</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
 
         <dependency>
             <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-common</artifactId>
-            <version>3.2.4</version>
+            <artifactId>hadoop-client-api</artifactId>
```

Comment:
The `hadoop-client-api` is a transitive dependency of the `hadoop-client-runtime.` We should remove the explicit definition of `hadoop-client-api`.
![grafik](https://github.com/apache/zeppelin/assets/454320/54374aaa-58d1-42e1-88c1-ee0123137102)

Reply:
I'm not sure if it's a good idea.
Technically, your suggestion is feasible and makes POM cleaner, but typically, we should explicitly declare the dependency that we consume directly.

Reply:
Yes, I understand the approach and can understand it. Thank you very much